concordium-std/src/impls.rs

```diff
@@ -606,17 +656,17 @@ where
 
     pub fn get(&self, key: K) -> Option<V> {
         let k = self.key_with_map_prefix(&key);
-        self.state_ll.borrow().lookup(&k).and_then(|mut entry| {
-            Some(V::deserial_state_ctx(&self.state_ll, &mut entry).expect_report(
+        self.state_ll.borrow().lookup(&k).map(|mut entry| {
+            V::deserial_state_ctx(&self.state_ll, &mut entry).expect_report(
```

Comment:
`expect_report` **cannot be used in production code**.

It is only available when testing locally and you **will not be able to build a contract that uses it**. So its use must be removed.